app/ante.go

```diff
@@ -48,14 +50,11 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		if options.GlobalFeeSubspace.Name() == "" {
 			return nil, sdkerrors.Wrap(sdkerrors.ErrNotFound, "globalfee param store is required for AnteHandler")
 		}
-		if options.StakingSubspace.Name() == "" {
-			return nil, sdkerrors.Wrap(sdkerrors.ErrNotFound, "staking param store is required for AnteHandler")
-		}
 
 		feeChecker := feechecker.NewFeeChecker(
 			options.OracleKeeper,
 			options.GlobalFeeSubspace,
```

Comment:
please help update GlobalFee to have its own params

Reply:
ok https://github.com/bandprotocol/chain/pull/320